api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -225,6 +225,31 @@ void shouldSendEmailWhenYoungestChildTurnsFourInNextPaymentCycle() throws JsonPr
         verifyNoMoreInteractions(notificationClient);
     }
 
+    @Test
+    void shouldSendEmailWhenYoungestChildTurnsFourAndPregnantInNextPaymentCycle() throws JsonProcessingException, NotificationClientException {
```

Comment:
I think it might be worthwhile parameterising the original test with the pregnancy date, the two tests are then identical apart from one assertion.

Reply:
Done